consensus/polybft/polybft_config.go

```diff
@@ -245,13 +245,17 @@ type GovernanceConfig struct {
 	VotingPeriod *big.Int
 	// ProposalThreshold indicates number of vote tokens required in order for a voter to become a proposer
 	ProposalThreshold *big.Int
+	// GovernorAdmin is the address of governance contract admin
+	// (he is the only one able to add new executors and proposers)
```

Comment:
Can the governor admin add new proposers and executors or add and remove them? Reading the PR description, I got the impression that it is able to both add and remove them. If that is the case, update the comment accordingly.

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1728/commits/834a98adb036dc64e915da3b97cc18b4c190aa5a